service/src/main/kotlin/fi/espoo/evaka/reports/ServiceVoucherValueReport.kt

```diff
@@ -134,19 +159,29 @@ private fun getServiceVoucherValues(
             child.first_name AS child_first_name,
             child.last_name AS child_last_name,
             child.date_of_birth AS child_date_of_birth,
+            child_group.name as child_group_name,
             unit.id AS unit_id,
             unit.name AS unit_name,
             area.id AS area_id,
             area.name AS area_name,
             daterange(decision.valid_from, decision.valid_to, '[]') AS service_voucher_period,
             part.voucher_value AS service_voucher_value,
             part.co_payment AS service_voucher_co_payment,
-            part.service_coefficient AS service_voucher_service_coefficient
+            part.service_coefficient AS service_voucher_service_coefficient,
+            part.hours_per_week service_voucher_hours_per_week
         FROM voucher_value_decision decision
         JOIN voucher_value_decision_part part ON decision.id = part.voucher_value_decision_id
         JOIN person child ON part.child = child.id
         JOIN daycare unit ON part.placement_unit = unit.id
         JOIN care_area area ON unit.care_area_id = area.id
+        LEFT JOIN LATERAL (
+            SELECT p.child_id, STRING_AGG(dg.name, ', ') AS name
+            FROM placement p
+            JOIN daycare_group_placement dgp ON p.id = dgp.daycare_placement_id
+            JOIN daycare_group dg ON dgp.daycare_group_id = dg.id
+            WHERE daterange(decision.valid_from, decision.valid_to, '[]') && daterange(dgp.start_date, dgp.end_date, '[]')
+            GROUP BY p.child_id, dgp.daycare_placement_id
```

Comment:
I think you should be grouping by `p.child_id` only. Also, it might be a good idea to only include placements to the placement unit on the voucher value decision, so adding something like `part.placement_unit = p.unit_id` to the `WHERE` clause.